src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -95,12 +114,92 @@ public void onCallback(final JSONObject options) {
             }
         }).timingOptions(timingCategory, timingVariableName, 0);
     }
+    
+    private void init() {
```

Comment:
For the finals / tabs vs spaces etc.  The whole project needs to be cleaned up to match other arcbees projects.

https://github.com/ArcBees/universal-analytics/issues/28